Benchmark-Models/model_info.py

```diff
@@ -22,6 +24,19 @@
 index_column = 'name'
 
 
+def get_number_of_events(petab_problem: petab.Problem) -> int:
+    """Check for events in the model"""
+    n_events = len(petab_problem.sbml_model.getListOfEvents())
+
+    parameter_ids = [p.getId() for p in
+                     petab_problem.sbml_model.getListOfParameters()]
+    for p_id in parameter_ids:
+        assignment_rule = petab_problem.sbml_model.getAssignmentRule(p_id)
+        if assignment_rule and 'piecewise(' in assignment_rule.getFormula():
+            n_events += 1
+    return n_events
```

Comment:
Maybe if `piecewise(` is seen in any expression, the events column value can have a `*` or other symbol added to it to indicate this.

Otherwise it might take a lot of parsing to get the "real" number of piecewise events. e.g. the following are currently missing from this method:
- piecewises in species assignments rules or other expressions (e.g. rate rules)
- the number of piecewises in a rule (e.g. a rule can have multiple piecewises -> multiple events)
- the number of conditions/pieces in each piecewise (i.e. each condition might be considered a separate event with a distinct trigger)